Assets/MixedRealityToolkit.Services/InputSimulation/Editor/InputSimulationService.cs

```diff
@@ -157,14 +157,8 @@ public override void LateUpdate()
                 // TODO implement custom hand device update frequency here, use 1000/fps instead of 0
                 if (msSinceLastHandUpdate > 0)
                 {
-                    if (HandDataLeft.Timestamp > lastHandUpdateTimestamp)
```

Comment:
**Note**: ManipulationHandlerThrow test is currently broken by this, because the way hand velocity is calculated relies on UpdateVelocity being called no more than once per frame.

Reply:
Fixed the throwing test. It was doing an additional WaitForFixedUpdate after releasing, which recomputes the hand velocity filtering and makes it deviate from the expected rigid body velocity. Previously this wasn't happening because the hand data update was blocked by the timestamps.